saleor/graphql/account/mutations/base.py

```diff
@@ -49,6 +49,7 @@ class SetPassword(CreateToken):
     account_errors = graphene.List(
```

Comment:
I think we can remove these fields (`user` and `account_errors`) because there are inherited from `CreateToken` mutation. 
We can move the description for `user` field to `CreateToken` mutation.